psl-core/src/main/java/org/linqs/psl/evaluation/statistics/RankingEvaluator.java

```diff
@@ -1,7 +1,7 @@
 /*
  * This file is part of the PSL software.
  * Copyright 2011-2015 University of Maryland
- * Copyright 2013-2021 The Regents of the University of California
+ * Copyright 2013-2020 The Regents of the University of California
```

Comment:
Old copyright.
You may need to pull your develop from upstream.

To pull from an upsteam fork, you can do:
```
git pull git@github.com:linqs/psl.git develop
```